### PR TITLE
New package: DeezloaderRemix-4.2.1

### PR DESCRIPTION
**Pull request by Gigahawk**

### CODE REVIEW

File: srcpkgs/DeezloaderRemix/template [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+do_build() {
+	npm install
+	npm run dist:linux
+}
+
+do_install() {
+	vmkdir usr/lib/deezloaderremix
+	vcopy dist/linux-unpacked/* /usr/lib/deezloaderremix/
+	vlicense LICENSE
```
Comment (Chocimier) [inline srcpkgs/DeezloaderRemix/template]: No need to attach GPL

File: srcpkgs/DeezloaderRemix/template [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+	vbin ${FILESDIR}/deezloader-rmx.sh deezloader-rmx
```
Comment (Chocimier) [inline srcpkgs/DeezloaderRemix/template]: Use symlink instead. It is simpler and doesn't cut off command line arguments.

File: srcpkgs/DeezloaderRemix/template [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+license="GPL-3.0-or-later"
+homepage="https://notabug.org/RemixDevs/DeezloaderRemix"
+distfiles="https://notabug.org/RemixDevs/DeezloaderRemix/archive/${version}.tar.gz"
+checksum=8469f2a6e23d478bf0938e518e8cabf1ddc47a37f780b0183ae5a3d2326bb896
+nocross=yes
+nostrip=yes
+shlib_provides="libnode.so libffmpeg.so"
+
+case "$XBPS_TARGET_MACHINE" in
+	i686*) broken="/void-packages/srcpkgs/DeezloaderRemix/template: line 17:  1928 Segmentation fault" ;;
```
Comment (Johnnynator) [inline srcpkgs/DeezloaderRemix/template]:
```suggestion
	i686*) broken="/void-packages/srcpkgs/DeezloaderRemix/template: line 17:  1928 Segmentation fault" ;;
	musl*) broken="DeezloaderRemix downloads against GLibc compiled binaries"
```

**Comment by Johnnynator** [comment]
Yuhu, yet another Electron package that pulls in binaries from outside the repo and doesn't compile them itself against our environment.  So first, the package should imo have a `-bin` Suffix.

**Comment by DirectorX** [comment]
We should not merge this one (in my opinion...). You rather use the `Appimage` they provide...

**Pull request closed without merging**